src/rabbit_mgmt_wm_operator_policies.erl

```diff
@@ -48,7 +48,7 @@ to_json(ReqData, Context) ->
       ["priority"], ReqData, Context).
 
 is_authorized(ReqData, Context) ->
-    rabbit_mgmt_util:is_authorized_admin(ReqData, Context).
```

Comment:
This handler doesn't serve `PUT` or `DELETE` requests, only `GET`.
We must show operator policies to management users, otherwise it would be confusing.